feature_layers/change-feature-layer-renderer/README.md

```diff
@@ -1,27 +1,34 @@
-# Change Feature Layer Renderer
+# Change feature layer renderer
 
-Change how a feature layer looks with a renderer.
+Change the appearance of a feature layer with a renderer.
 
-![](ChangeFeatureLayerRenderer.gif)
+![Image of change feature layer renderer](ChangeFeatureLayerRenderer.png)
+
+## Use case
```

Comment:
again, another example of a sample where the readme design uses different data to our Java sample (ours still uses the pool permits, not the trees). I logged an issue for this in the Common Samples repo (no. 1471). The updated service is the following if you wanted to fix it up as part of this, otherwise let's revert this to the original description until we update the data. 

https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/Landscape_Trees/FeatureServer/0

Reply:
Of course, totally happy to update the data!